integration-tests/gatsby-cli/__tests__/build-ssr-errors.js

```diff
@@ -0,0 +1,42 @@
+import { GatsbyCLI } from "../test-helpers"
+
+const MAX_TIMEOUT = 2147483647
+jest.setTimeout(MAX_TIMEOUT)
+
+describe(`gatsby build (SSR errors)`, () => {
+  const cwd = `gatsby-sites/gatsby-build-ssr-errors`
+
+  beforeAll(() => GatsbyCLI.from(cwd).invoke(`clean`))
+  afterAll(() => GatsbyCLI.from(cwd).invoke(`clean`))
+
+  it(`should log build errors with useful outputs`, () => {
+    const [code, logs] = GatsbyCLI.from(cwd).invoke(`build`)
+
+    // Generic logging header
+    logs.should.contain(`failed Building static HTML for pages`)
```

Comment:
could we move to inlineSnapshot or would that be too wild?

Reply:
snapshots do not work because the logged output can include things like file paths, which change on every persons machine.